docs/releases/slashing-release/interfaces/core/slashing-parameter-configuration.md

```diff
@@ -0,0 +1,222 @@
+     * @param operatorSet the operatorSet to check slashability for
+     * @return bool if the operator is slashable by the operatorSet
+     */
+    function isOperatorSlashable(address operator, OperatorSet memory operatorSet) external view returns (bool);
+
+}
+```
+
+### modifyAllocations
+
+Operators call this to set the proportions of slashable stake allocated to a list of operatorSets for a set of strategies. Depending on what the new magnitude/proportion and current magnitude value is configured to, this will either create a pending allocation or deallocation. Allocations by default have a delay of 21 days but can be one-time configured by an operator. Deallocation delays all have a built-in delay of 17.5 days until they can be "completed". There is a "completing" second step to deallocations that must take place to account for the fact that deallocations are still susceptible to slashing by the operatorSet up until the completeableTimestamp for the deallocation has passed. Pending allocations on the other hand are not slashable (referring to the added increase in magnitude).
+
+Note that allocation delays are more of a concern to stakers as it is increasing the risk to an operator's delegated stakers. Thus they may need enough time to undelegate and withdraw their stake from an operator if they do not agree with a pending allocation. However, some operators are also their own staker (ex. LRTs) and may not care about this allocation delay so they could configure it to be 0.
+
+Deallocation delays are important for AVSs and their operatorSets to ensure they can have a minimum future forecast of slashable stake so that all of their economic security cannot leave their system instantly.
+
+Operators call this to allocate to their slashable stake (magnitudes) for a list of (operator, IStrategy, operatorSet(avs, operatorSetId)) tuples. For each MagnitudeAllocation param given, it has a given IStrategy and it queues magnitude updates to the specified operatorSets which will take effect 21 days from the time of calling. This gives the operator's stakers 3.5 days to queue withdrawals if they disagree with the changes to their staking portfolio.
+
+All allocations in the call are summed and checked to be less than the free magnitude available for allocation for the `strategy`. This free magnitude amount is nonslashable by an operatorSet. This is in order for all allocations to be backed stake that will be slashable when the update takes effect.
+
+The function can be called with an EIP1271 signature from the operator or by the operator itself.
+
+Emits
+
+1. if a `MagnitudeAllocation` results in a value greater than the current magnitude, `MagnitudeAllocated` is emitted for the given (operator, IStrategy, operatorSet)
+2. if a `MagnitudeAllocation` results in a value less than the current magnitude,
+`MagnitudeDeallocated` is emitted for the given (operator, IStrategy, operatorSet)
+3. 
```

Comment:
this looks messed up

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/709/commits/25d410df3f01744b5a1ca30dda06d7750a5e5751
Take 2?